randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -1031,6 +1053,24 @@ morph gliding come to mind here.
       Speed Booster and Shinesparking Tricks (Intermediate)
   > Pickup (Super Missile Tank)
       Super Missiles and Power Grip Climb and Tunnel Climb
+  > Top right
+      Any of the following:
+          All of the following:
+              # Morph Glide
+              Hi-Jump Boots and Morph Ball
+              Any of the following:
+                  Morph Glide (Advanced)
+                  All of the following:
+                      # This makes gliding not as strict
+                      Morph Glide (Intermediate)
+                      Power Grip or Walljump (Beginner)
+          All of the following:
+              # Speedboost way
+              Speed Booster
+              # Charge Speedboost
+              Movement (Intermediate) or Short Charge (Beginner)
```

Comment:
Wondering what the movement is here.

Reply:
You can get a speedboost going, but it requires some precise movement to moonwalk right at the edge of the door so that you don't trigger the transition.